src/EA.Weee.Web/Views/Shared/_EvidenceReportHintText.cshtml

```diff
@@ -1 +1,3 @@
-﻿<p class="govuk-details__text">The data shown in the table is for household evidence.<br />The evidence column includes approved evidence and net transfer.</p>
```

Comment:
@meglenapnovoselska I think this hint text maty be used elsewhere. May be worth pulling this hint text out so it is specific to the PCS.

Reply:
okay, I will